OperatingSystem-Services/Service-SOC/BasicDocumentation/Wazuh-Docker-Single-Node.md

```diff
@@ -0,0 +1,20 @@
+# How to Deploy Wazuh on Docker, Single-Node
+
+
```

Comment:
Could add something about running this on an Ubuntu system for example. Just explain your environment. That could help out.